bentoml/_internal/external_typing/__init__.py

```diff
@@ -20,6 +20,8 @@
 
     NpNDArray = _NDArray[t.Any]
 
+    from xgboost import DMatrix
```

Comment:
does this mean developers have to install xgboost, in order to run type checking?

Reply:
If they work on any files that use it, yeah.